build.gradle

```diff
@@ -1,9 +1,9 @@
 plugins {
     id 'java'
+    id 'com.adarshr.test-logger' version '3.2.0'
```

Comment:
We don't need this plugin, do we?

Reply:
This make the test output more readable. But we don't need it, reverted.